advanced_functionality/pytorch_deploy_pretrained_bert_model/pytorch_deploy_pretrained_bert_model.ipynb

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
<p>You mention deleting the endpoint configuration - either add in the config deletion or remove the mention of it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Removed the mention of it since it wasn't really needed.